.github/workflows/msktutil-macos.yml

```diff
@@ -0,0 +1,36 @@
+name: msktutil macOS
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build on macOS
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg:
+          - { cc-version: gcc }
+          - { cc-version: clang }
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Build msktutil ${{ matrix.cfg.cc-version }}
+      shell: sh
+      run: |
+        case "${{ matrix.cfg.cc-version }}" in
+          gcc)
+            export CC=gcc
```

Comment:
Where does gcc come from? It does not come preinstalled from Apple.

Reply:
yes, I found out everything is already preinstalled there.